pandas/core/arrays/boolean.py

```diff
@@ -281,7 +281,9 @@ def __getitem__(self, item):
             return self._data[item]
         return type(self)(self._data[item], self._mask[item])
 
-    def _coerce_to_ndarray(self, force_bool: bool = False):
+    def _coerce_to_ndarray(
+        self, force_bool: bool = False, na_value: "Scalar" = lib._no_default
```

Comment:
Any reason to prefer `lib_no_default` to just `libmissing.NA` directly?

Reply:
Hmm, not sure. I was probably sidetracked by the idea I could not use None as default as that is a valid value as well .. 
(if we want to have this generic / shared with other arrays, using `self._na_value` might be useful, but I don't think we will share this with arrays that don't use pd.NA, so ..)